app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
```

Comment:
When iterate through lists we usually not refer to items with indexes. Actually we don't need index of each friend, we need friend object itself on each iteration. So we should iterate using this approach:
```suggestion
    for friend in friends:
```